README.md

```diff
@@ -25,10 +14,22 @@ Perform the actions from the sub-documents in the following order:
 
 * [Copy MariaDB data](mariadb_copy.md)
```

Comment:
Before `MariaDB` copy, we can have a prereq section where we assume TripleO is up && running on one side, and the basic ctlplane is running on the other side: the first thing to do is run crc with all the `disabled` services.
iirc this last topic is covered by [Deploy OpenStack control plane services](https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/backend_services_deployment.md), and it should go first, right?

Reply:
That's pretty much all in "Deploy podified backend services" which is already the step right before MariaDB copy. I could expand the prereq list to mention that we need to have the cloud we want to Adopt up and running.

Reply:
Thanks for clarifying! LGTM.

Reply:
I'll add one more commit mentioning that the cloud which we want to adopt should be up and running, and it has to be on Wallaby. And i'll merge.